styles/popup.css

```diff
@@ -0,0 +1,198 @@
+@media screen and (max-width: 992px) {
+  .popup-content {
+    display: none;
+  }
+
+  .popup-body {
+    background: #fff;
+    position: fixed;
+    top: 5%;
+    left: 2.5%;
+    overflow-y: scroll;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin: 0 20px 40px 0;
+    padding: 16px;
+    height: 90vh;
+  }
```

Comment:
- [ ] Please make sure to add `blury` background around the popup window like given in Figma: 

Yours  | Figma
------------- | -------------
![b22](https://user-images.githubusercontent.com/57562869/152518189-390f7357-fda4-4718-8090-17ddb98fa94b.png) | ![g22](https://user-images.githubusercontent.com/57562869/152518245-a7e6e4ab-571b-410d-8133-ea90617980a5.png)

As you can see the blurry background in Figma serves for a better focus on popup content 😉 


- [ ] Please give to the body { overflow: hidden } when the popup is visible, to prevent scrolling the page because currently, users can scroll the screen which should not be the case 😢